ODATA-1432 (#134)
oasis-tcs/odata-specs: odata-csdl/7 Structural Property.md

## ##subsec Type Facets

> Facets modify or constrain the acceptable values for a model element of a given type.

Reviewer: ```suggestion
Facets applied to a property modify or constrain the acceptable values for the property.
```

> For single-valued model elements the facets apply to the value of the

Reviewer: As this is within the definition of a property, it seems we can be clearer by using "property" here.
